reference/forms/types/entity.rst

```diff
@@ -202,9 +202,10 @@ query_builder
 Allows you to create a custom query for your choices. See
 :ref:`ref-form-entity-query-builder` for an example.
 
-The value of this option can either be a ``QueryBuilder`` object, a Closure or null.
+The value of this option can either be a ``QueryBuilder`` object, a Closure or ``null``.
 When using a Closure, you will be passed the ``EntityRepository`` of the entity
-as the only argument and should return a ``QueryBuilder``.  If you'd like to display a list of empty entries, you can return null in the query_builder closure.
+as the only argument and should return a ``QueryBuilder``.  
+If you'd like to display a list of empty entries, you can return ``null`` in the closure.
```

Comment:
@xabbuh Looks like https://github.com/symfony/symfony/commit/317d30b6efd4e906e9c84fbe558934448379285c has not made it to the core !? (neither [master](https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php#L50) nor [2.8](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php#L84))